client/src/components/MenuItem.js

```diff
@@ -69,11 +68,12 @@ function MenuItem(props) {
 
     const purchaseMeal = (e) => {
         e.preventDefault();
-        const id = parseFloat(e.currentTarget.value);
+        const id = e.currentTarget.value;
+        const chefID = chefId;
+        const meal = { id, picURL, title, price, chefName, chefID };
```

Comment:
And why did we move this declaration up a few lines?

Reply:
I had rename a few values that were passed to the cart because they were bit different once we started getting the meal from the DB.  I believe I moved just to make it easier to work with at the time.

Reply:
I had rename a few values that were passed to the cart because they were bit different once we started getting the meal from the DB.  I believe I moved just to make it easier to work with at the time.

Reply:
We only use it inside the else statement, so I think we should define it there